event_operating_unit/views/event_track_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_event_track_tree_inherit_ou">
+        <field name="name">event.track.tree</field>
+        <field name="model">event.track</field>
+        <field name="inherit_id" ref="website_event_track.view_event_track_tree" />            
+        <field name="arch" type="xml">
+            <tree>
```

Comment:
Inheritance is wrong here as `name`and `active` are already defined in `website_event_track.view_event_track_tree`
Please use `<field name="" position="">` to add only `operating_unit_id`